packages/core/src/app/shipping/shippingOption/ShippingOptionsForm.scss

```diff
@@ -5,13 +5,13 @@
     border: container("border");
     border-radius: $global-radius;
     display: flex;
-    min-height: 10rem;
-    padding: spacing("half");
+    min-height: 9.383rem;
```

Comment:
Reduce the shipping options default height to the same as the loading skeleton's height.
<img width="573" alt="image" src="https://user-images.githubusercontent.com/88361607/196093748-be1365db-73aa-41bc-9f16-2c6e6229a70a.png">